simpeg_drivers/depth_of_investigation/sensitivity_cutoff/driver.py

```diff
@@ -0,0 +1,82 @@
+# ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+#  Copyright (c) 2024 Mira Geoscience Ltd.
+#  All rights reserved.
+#
+#  This file is part of simpeg-drivers.
+#
+#  The software and information contained herein are proprietary to, and
+#  comprise valuable trade secrets of, Mira Geoscience, which
+#  intend to preserve as trade secrets such software and information.
+#  This software is furnished pursuant to a written license agreement and
+#  may be used, copied, transmitted, and stored only in accordance with
+#  the terms of such license and with the inclusion of the above copyright
+#  notice.  This software and information or any other copies thereof may
+#  not be provided or otherwise made available to any other person.
+#
+# ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+import logging
+import sys
+
+import numpy as np
+from geoapps_utils.driver.driver import BaseDriver
+from geoh5py.data import FloatData
+from geoh5py.objects import Octree
+from geoh5py.shared.utils import fetch_active_workspace
+
+from .params import SensitivityCutoffParams
+
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+def scale_sensitivity(sensitivity: FloatData) -> np.ndarray:
+    """
+    Normalize sensitivities by cell sizes and scale as percentage.
+
+    :param sensitivity: Sum squared sensitivity matrix.
+    """
+    mesh = sensitivity.parent
+    cell_sizes = (
+        mesh.octree_cells["NCells"]
+        * mesh.u_cell_size
+        * mesh.octree_cells["NCells"]
+        * mesh.v_cell_size
+        * mesh.octree_cells["NCells"]
+        * mesh.w_cell_size
+    )
```

Comment:
Can we add a `cell_volumes` attribute to the Octree?

Reply:
https://github.com/MiraGeoscience/geoh5py/pull/667

Reply:
Ok but then we need to remove still here, or we change the directive to normalize by volumes.

Reply:
yep, forgot.  done now.